### DISCUSSION TITLE
How do I set md math codes to LaTeX font in qmd?

### DISCUSSION THREAD

**Discussion by Kaprekar2** [quarto-dev/quarto-cli#9791] [Q&A] [answered]
### Description

I am using quarto presentation formatted by revealjs. When I write and preview markdown math codes, it renders to Roman font, but after it rendered, it looks rendered other fonts rather than Roman font like below.

Please let me teach if you know why and (if it could) how to resolve it.

OS: Sonoma 14.2.1
Rstudio: 2023.12.1+402
quarto: 1.3.361
Mathjax: 3

![preview](https://github.com/quarto-dev/quarto-cli/assets/52351403/b347c0ac-f531-4da5-8e8e-d4881278e65c)
![rendered](https://github.com/quarto-dev/quarto-cli/assets/52351403/1db1bbd9-1e09-4ad5-893b-93691015e0fb)

**Comment by cderv** [comment, marked as answer]
Did you try `mathfont` and `mathfontoptions` config already ? 
https://quarto.org/docs/reference/formats/pdf.html#fonts

This a LaTeX problems: If you find how to tweak mathfont for LaTeX, it should work for quarto.

**Reply by Kaprekar2** [reply]
Thank you for your comment! (and I'm sorry for my reply delaying)

This rendering problem is for MathJax rather than Quarto md.
If user doesn't explicitly specify MathJax in yaml frontmatter, `html-math-method: mathjax`, it seems to render Gothic-like font on Chrome.
(On FireFox and Safari, without the syntax, it rendered to roman font properly.)

・reference
https://github.com/quarto-dev/quarto-cli/discussions/3127
